ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml

```diff
@@ -265,6 +265,17 @@
                         <jcr:content jcr:primaryType="nt:unstructured"/>
                     </policy_1588584656785>
                 </container>
+				<teaser jcr:primaryType="nt:unstructured">
+                    <policy_1588926494157
+                        jcr:lastModified="{Date}2020-05-08T15:27:31.381+05:30"
+                        jcr:lastModifiedBy="admin"
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Teaser policy"
+                        sling:resourceType="wcm/core/components/policy/policy"
+                        titleLinkHidden="true">
```

Comment:
@Rajamouli-hcl : why do we need titleLinkHidden in policy?

Reply:
when we select this the teaser title is not link to the linked page

Reply:
@Rajamouli-hcl : Title wont be linked until a URL is specified in dialog . Is it not the case ? I removed this policy and still title is not a link unless URL added

![image](https://user-images.githubusercontent.com/38690824/81469465-9247f280-9202-11ea-8110-28fbe51aeeb7.png)

Reply:
yes correct rishabh, i thought this story  even if the url is specified in dialog that won't be linked with title for this i added policy,if it's not the case please confirm i will remove policy

Reply:
@Rajamouli-hcl : Pl remove this policy

Reply:
policy has been removed as per the review comments